app/bundles/CoreBundle/Assets/js/1a.content.js

```diff
@@ -979,33 +980,45 @@ Mautic.activateChosenSelect = function(el, ignoreGlobal, jQueryVariant) {
             singlePlaceholder = mauticLang['chosenChooseOne'];
     }
 
-    if (typeof mQuery(el).data('chosen-placeholder') !== 'undefined') {
-        multiPlaceholder = singlePlaceholder = mQuery(el).data('chosen-placeholder');
+    if (typeof $select.data('chosen-placeholder') !== 'undefined') {
+        multiPlaceholder = singlePlaceholder = $select.data('chosen-placeholder');
     }
 
-    mQuery(el).chosen({
+    const hide_results_on_select = !$select.attr('data-chosen-keep-results-on-select');
```

Comment:
for consitency in variable naming
```suggestion
    const hideResultsOnSelect = !$select.attr('data-chosen-keep-results-on-select');
```